<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsEditorGeneral</class>
 <widget class="QWidget" name="SettingsEditorGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../data/icons.qrc">
    <normaloff>:/icons/48x48/settings.png</normaloff>:/icons/48x48/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>370</width>
        <height>345</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelLogoEditor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="pixmap">
            <pixmap resource="../../../data/logo.qrc">:/logo/48x48/logo-editor.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelVersion">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupLanguage">
         <property name="title">
          <string>Language</string>
         </property>
         <layout class="QVBoxLayout" name="layoutLangBox">
          <item>
           <widget class="QRadioButton" name="radioDefault">
            <property name="text">
             <string>System default language</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioCustom">
            <property name="text">
             <string>Custom language</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboLanguage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupDefaults">
         <property name="title">
          <string>Defaults</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonDefaults">
            <property name="text">
             <string>Restore defaults</string>
            </property>
            <property name="icon">
             <iconset resource="../../../data/icons.qrc">
              <normaloff>:/icons/16x16/refresh.png</normaloff>:/icons/16x16/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelDefaults">
            <property name="text">
             <string>&lt;i&gt;This will restore all your settings to default. You need to apply the changes, before they will take effect.&lt;/i&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
  <include location="../../../data/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboLanguage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
